### PR TITLE
Milestone2: Components

### PR DESCRIPTION
**Pull request by miftah1991**
in this milestone bellow features have been added:

- Deleted all the boilerplate from CRA (text, images, styles).
- Created a directory called components.
- Created a new Calculator.jsx file.
- Created a React component that matches the design.
- Used class-based components.
- No `linters `error

### CODE REVIEW

File: src/App.test.js [hunk 1]
```diff
@@ -1,5 +1,5 @@
 import { render, screen } from '@testing-library/react';
-import App from './App';
+import App from './components/App';
```
Comment (Gambit142) [inline src/App.test.js]: - [ ] Kindly delete this file. Remember that part of the project requirements is for you to delete all _**boilerplates**_ created with _**Create React App (CRA)**_

File: src/components/Calculator.jsx [hunk 2]
```diff
@@ -0,0 +1,117 @@
...
+          </Col>
+          <Col>
+            <Button className="w-100 p-4" variant="warning">
+              =
+            </Button>
+          </Col>
+        </Row>
+      </Container>
+    );
+  }
```
Comment (Gambit142) [inline src/components/Calculator.jsx]: - [ ] Kindly style your calculator to loo like that of the template given. Kindly use colours similar to that used in the template.

### Your Design

image

### Calculator Template

image

**Review by Gambit142 — CHANGES_REQUESTED** [review]
Hi @miftah1991 👋, 

Good job so far!

### Highlights 
- Linters Checks passed 💯 
- Good Use of class-based Components 💯 

However, there are some issues that you still need to work on to go to the next project but you are almost there!

## Required Changes ♻️
- Kindly delete the following files that were created when you ran the _**Create React App**_ command 
  - App.css
  - reportWebVitals.js
  - setUpTests.js
Remember part of the project requirements is for you to delete all boilerplates created with _**CRA**_. I could not use inline comments for them because they were not pushed. 

_Kindly check the comments under the review for further changes to be made._

Cheers and Happy coding!👏👏👏

Feel free to leave any questions or comments in the PR thread if something is not 100% clear.
Please, remember to tag me in your question so I can receive the notification.

**_Please, do not open a new Pull Request for re-reviews. You should use the same Pull Request submitted for the first review, either valid or invalid unless it is requested otherwise._**

------
_As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561) you have a limited number of reviews per project (check the exact number in your Dashboard). If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s)._
```

File: src/index.css [hunk 3]
```diff
@@ -1,27 +1,33 @@
-body {
+* {
```
Comment (Meltrust) [inline src/index.css:1]: - [ ] As requested by the previous reviewer, please make your calculator _match_ the given design.

In this review iteration, you are mostly matching the colors, but not the widths, the alignments, the padding, the margins.


Your Implementation | Given Design
:---:|:---:
image | image

I recommend using flexbox to align your calculator, so it's easier.

Please, kindly make your calculator match the given design.

**Review by Meltrust — CHANGES_REQUESTED** [review]
Hi @miftah1991,

Great work on making the changes requested by a previous reviewer 👏 
You've done well implementing some of the requested changes, but there are still some which aren't addressed yet.

You can do it

 ## Required Changes ♻️
_Check the comments under the review._

## Optional suggestions

_Every comment with the [OPTIONAL] prefix won't stop the approval of this PR. However, I strongly recommend you to take them into account as they can make your code better. Some of them were simply missed by the previous reviewer and addressing them will really improve your application._


Cheers and Happy coding!👏👏👏

Feel free to leave any questions or comments in the PR thread if something is not 100% clear.
Please, remember to tag me in your question so I can receive the notification.

**_Please, do not open a new Pull Request for re-reviews. You should use the same Pull Request submitted for the first review, either valid or invalid unless it is requested otherwise._**


------
_As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561)  you have a limited number of reviews per project (check the exact number in your Dashboard). If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s)._

**Review by Meltrust — APPROVED** [review]
Hi @miftah1991,


## Wow, you did it 🎉
Brilliant
## Thank you for correcting y your calculator design, as requested in the previous review💪 🥇 ㊗️ 
### Well done!




Your project is complete!  There is nothing else to say other than... it's time to merge it :shipit: 
Congratulations! 🎉 



Cheers and Happy coding!👏👏👏

Feel free to leave any questions or comments in the PR thread if something is not 100% clear.


------
_As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561)  you have a limited number of reviews per project (check the exact number in your Dashboard). If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s)._

**Comment by miftah1991** [comment]
@Meltrust ,

Thank you for your review.

**Pull request merged**